examples/openthread/Makefile

```diff
@@ -4,7 +4,31 @@ APPLICATION = openthread
 BOARD ?= samr21-xpro
 
 # These are the boards that OpenThread stack has been tested on
-BOARD_WHITELIST := samr21-xpro iotlab-m3 fox iotlab-a8-m3 frdm-kw41z openlabs-kw41z-mini-256kib openlabs-kw41z-mini phynode-kw41z usb-kw41z
+BOARD_WHITELIST := \
+					samr21-xpro \
+					iotlab-m3 \
+					fox \
+					iotlab-a8-m3 \
+					frdm-kw41z \
+					openlabs-kw41z-mini-256kib \
+					openlabs-kw41z-mini \
+					phynode-kw41z \
+					usb-kw41z \
+					cc2538dk \
+					remote-reva \
+					remote-revb \
+					omote \
+					openmote-b \
+					openmote-cc2538 \
+					adafruit-clue \
+					adafruit-itsybitsy-nrf52 \
+					arduino-nano-33-ble \
+					feather-nrf52840 \
```

Comment:
Those boards should not be part of this PR because they don't work with openthread (no stdio on uart).
```suggestion
```